### DISCUSSION TITLE
Python roadmap

### DISCUSSION THREAD

**Discussion by chia-kai-chang** [1101-GS3073-C/Python-Community-GS3073-C#17] [General]
![image](https://user-images.githubusercontent.com/1768891/135586335-f6960a57-5616-4411-8831-c74155519f01.jpeg)

Source: https://www.linkedin.com/in/mahmoud-essam-822719149

**Comment by boson13579** [comment]
Roadmap 就是學習路線圖喔~給不知道的說一下，雖然我猜大家都還卡在很前面XD，應該跟我一樣只認識圖上一點點東西而已XD
